notebook/static/notebook/js/searchandreplace.js

```diff
@@ -163,6 +163,7 @@ define([
 
       var allCellsButton = $('<button/>')
       .append($('<i/>').addClass('fa fa-arrows-v'))
+      .attr('id', 'allcells_id')
```

Comment:
IDs don't need to have a `_id` suffix, but they do need to be unique on the whole page - that includes IDs that might come from extensions or from output produced in a notebook. So let's make them a bit more specific, like `findreplace_allcells_btn` - and likewise for the other ones added here.